### ISSUE TITLE
Switch template: error message is wrong when using CoA

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#5166]
**Describe the bug**
When you use CoA to reevaluate access using a switch template, you got a **wrong** error messages in logs:

```
Feb 27 14:12:22 pfcen7night pfqueue: pfqueue(8231) WARN: [mac:77:77:77:77:77:77] Unable to perform RADIUS Disconnect-Request: No answer from 192.168.0.6 on port 3799 at /usr/local/pf/lib/pf/util/radius.pm line 147. (pf::Switch::Template::catch {...} )
```

**Expected behavior**
Log message must be: `Unable to perform RADIUS CoA-Request`.

**Additional context**
I see other logs messages in https://github.com/inverse-inc/packetfence/blob/devel/lib/pf/Switch/Template.pm that need to be changed.

**Comment by jrouzierinverse** [comment]
Was this fixed in #5164?

**Comment by nqb** [comment]
@jrouzierinverse, I suppose your last comment was not related to this issue.

**Comment by jrouzierinverse** [comment]
Fixed by d6bfb098ad9d8d4a9a733b6f32fd974076580c9a
